Utils/python/prescaleweightproducer_cfi.py

```diff
@@ -6,6 +6,6 @@
 
    prescales = cms.InputTag("patTrigger"),
 
-   objects = cms.InputTag("selectedPatTrigger"),
+   objects = cms.InputTag("slimmedPatTrigger"),
```

Comment:
this change also breaks 2016 compatibility, so please revert it if possible or remove all 2016 scenarios and input file lists (including from README, unit tests)